.jenkins/pytorch/win-test-helpers/build_pytorch.bat

```diff
@@ -96,7 +96,7 @@ if "%USE_CUDA%"=="1" (
   :: so we are actually pretending sccache instead of nvcc itself.
   curl -kL https://github.com/peterjc123/randomtemp-rust/releases/download/v0.3/randomtemp.exe --output %TMP_DIR_WIN%\bin\randomtemp.exe
   set RANDOMTEMP_EXECUTABLE=%TMP_DIR_WIN%\bin\nvcc.exe
-  set CUDA_NVCC_EXECUTABLE=%TMP_DIR_WIN%\bin\randomtemp.exe
+  set CUDA_NVCC_EXECUTABLE=%TMP_DIR%/bin/randomtemp.exe
```

Comment:
I think this is issue might be cygwin related? It was interpreting the path as unix-style and so the backslashes were parsed as escape characters.